nixos/modules/programs/captive-browser.nix

```diff
@@ -0,0 +1,108 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.programs.captive-browser;
+in
+{
+  ###### interface
+
+  options = {
+    programs.captive-browser = {
+      enable = mkEnableOption "captive browser";
+
+      package = mkOption {
+        type = types.package;
+        default = pkgs.captive-browser;
```

Comment:
``warning: Option `programs.captive-browser.package' has no description.``

Reply:
Additionally, please provide `defaultText = "pkgs.captive-browser";` so that it's more readable in the manual and it also gets evaluated IFF it's actually enabled.

Reply:
Fixed with #63135